README.md

```diff
@@ -75,19 +75,51 @@ Optional. A string with a BCP 47 language tag, or an array of such strings. For
 
 Optional. An object with some or all of the following properties:
 
-##### `localeMatcher`
+* ##### `localeMatcher`
```

Comment:
I'd still prefer this option to be deprioritized and put last, since it's the least useful, even if it's alphabetically first.